zaas-service/src/test/resources/zaas-messages.yml

```diff
@@ -229,3 +229,18 @@ messages:
       text: "z/OSMF is not available or z/OSMF response does not contain any token. Reason: %s"
       reason: z/OSMF does not return JWT or LTPA tokens.
       action: Make sure z/OSMF is available to API ML or review your z/OSMF configuration.
+
+    - key: org.zowe.apiml.common.notFound
+      number: ZWEAO404
```

Comment:
What is ZWEAO?

Reply:
Also as a reminder to confirm these go to the zowe docs

Reply:
I just copy them from application resource